packages/@aws-cdk/aws-lambda/lib/runtime.ts

```diff
@@ -223,7 +223,7 @@ export class Runtime {
     this.name = name;
     this.supportsInlineCode = !!props.supportsInlineCode;
     this.family = family;
-    const imageName = props.bundlingDockerImage ?? `amazon/aws-sam-cli-build-image-${name}`;
+    const imageName = props.bundlingDockerImage ?? `public.ecr.aws/sam/build-${name}`;
```

Comment:
Do you happen to know the difference between the `sam/build-` images and the ones provided by the AWS Lambda vendor - https://gallery.ecr.aws/?page=1&searchTerm=%22aws+lambda%22 ?

Reply:
The `public.ecr.aws/lambda/*` are runtime images. They don't have all the tools required to build/install dependencies.

Reply:
Got it.